react-ui/web/src/components/analysis/integrated_site/BackToMainButton.js

```diff
@@ -11,18 +11,164 @@ const goBack = history => e => {
     history.replace("/analysis/sites");
 };
 
+const date_range_options = [
+    { label: "3 days", unit: "days", duration: 3 },
+    { label: "7 days", unit: "days", duration: 7 },
+    { label: "10 days", unit: "days", duration: 10 },
+    { label: "2 weeks", unit: "weeks", duration: 2 },
+    { label: "1 month", unit: "months", duration: 1 },
+    { label: "3 months", unit: "months", duration: 3 },
+    { label: "6 months", unit: "months", duration: 6 },
+    { label: "1 year", unit: "years", duration: 1 },
+    { label: "All", unit: "all", duration: 0 }
+];
+
+const hour_interval = [
+    { label: "4 hours", hour_value: "4" },
+    { label: "8 hours", hour_value: "8" },
+    { label: "12 hours", hour_value: "12" },
+    { label: "1 day", hour_value: "24" }
+];
+
 function BackToMainButton (props) {
-    const { width, history } = props;
+    const { width, history, chartType,
+        selected, setSelected, setSelectedRangeInfo,
+        selected_hour_interval, setSelectedHourInterval } = props;
+    const [ dateRangeAnchor, setDateRangeAnchor ] = useState(null);
+    const [ hourIntervalAnchor, setHourIntervalAnchor ] = useState(null);
```

Comment:
Remember na kapag in-file state, snake case tayo. Kapag property, yung camel case.
Change following variables accordingly.
```suggestion
    const [ date_range_anchor, setDateRangeAnchor ] = useState(null);
    const [ hour_interval_anchor, setHourIntervalAnchor ] = useState(null);
```